test/crypto/eax.js

```diff
@@ -82,17 +82,17 @@ function testAESEAX() {
         nonce: "22E7ADD93CFC6393C57EC0B3C17D6B44",
         header: "126735FCC320D25A",
         ct: "CB8920F87A6C75CFF39627B56E3ED197C552D295A7CFC46AFC253B4652B1AF3795B124AB6E"
-      },
+      }
     ];
 
     const cipher = 'aes128';
 
-    for(const [i, vec] of vectors.entries()) {
-      const keyBytes = openpgp.util.hex_to_Uint8Array(vec.key),
-        msgBytes = openpgp.util.hex_to_Uint8Array(vec.msg),
-        nonceBytes = openpgp.util.hex_to_Uint8Array(vec.nonce),
-        headerBytes = openpgp.util.hex_to_Uint8Array(vec.header),
-        ctBytes = openpgp.util.hex_to_Uint8Array(vec.ct);
+    vectors.forEach(async vec => {
```

Comment:
You have to
```js
    await Promise.all(vectors.map(async vec => {
```
in order to make sure all of them succeed (in test/crypto/ocb.js as well).

Other than that the PR looks good :+1: